app/Services/Plugins/ImporterService.php

```diff
@@ -389,11 +389,9 @@ protected function importImage(MiscModel $model, PluginVersionEntity $entity)
 
```

Comment:
The if $model->image makes no sense now that models no longer have this property